pom.xml

```diff
@@ -265,6 +265,11 @@
     	<version>2.4</version>
     </dependency>
     <dependency>
+      <groupId>commons-dbutils</groupId>
+      <artifactId>commons-dbutils</artifactId>
+      <version>1.6</version>
+    </dependency>
```

Comment:
Please add XML comments explaining what the dependencies are.

Reply:
Done, and some dependencies removed entirely.